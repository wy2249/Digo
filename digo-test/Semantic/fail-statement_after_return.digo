func digo_main() void {
  foo()
}

func foo() int {
  return 0
  return 10
}