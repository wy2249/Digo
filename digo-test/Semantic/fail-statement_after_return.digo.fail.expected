Fatal error: exception Failure("Statements appear after Return")
